tools/packaging/static-build/cache_components_main.sh

```diff
@@ -55,6 +55,7 @@ cache_nydus_artifacts() {
 cache_ovmf_artifacts() {
 	local current_ovmf_version="$(get_from_kata_deps "externals.ovmf.${OVMF_FLAVOUR}.version")"
 	[ "${OVMF_FLAVOUR}" == "tdx" ] && OVMF_FLAVOUR="tdvf"
+	[ "${OVMF_FLAVOUR}" == "sev" ] && OVMF_FLAVOUR="ovmf-sev"
```

Comment:
The caching for ovmf works fine. I ran `make ovmf-sev-tarball` on  my workstation, it picked up the tarball from Jenkins as expected. So this commit is ready to go.